CHANGELOG.md

```diff
@@ -12,6 +12,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - added new method `Grid::delete_orders` and the corresponding switch
   `--delete-orders` in the subcommand `write` of the CLI
 
+### Changed
+
+- dropped top-level Python interface layer
+- renamed `lumi` to `channel` in PyO3 Python interface. This concerns 1) the argument names of `convolute_with_one` and similar functions; 2) the module `Python API:pineappl.lumi` was moved to `Python API:pineappl.channel`; 3) the class `LumiEntry` was renamed to `Channel`
+- `.into()` needs to be explicitly called on subgrids when calling `Python API:pineappl.grid.set_subgrid()`
+- replaced `Python API:pineappl.grid.PyPidBasis` with `Python API:pineappl.evolution.PidBasis`
```

Comment:
sorry, what I meant was something along the lines of

- Python API: replaced `pineappl.grid.PyPidBasis` with `pineappl.evolution.PidBasis`

Reply:
Also manually break the lines after a maximum of 80 characters.

Reply:
Also addressed in [230b188](https://github.com/NNPDF/pineappl/pull/302/commits/230b188c48df1d817e7dbf949f74d271356de50f).